dist/docker/fuzz-asm/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
```

Comment:
Maybe pin OS version? e.g. jammy

Reply:
im always confused about ubuntu/debian release names, i prefer version numbers, but yeah both dockerfiles can be improved, but i dont think the version of the system really affects much in the fuzzing process. the latest the better i guess because that will end up building with a more modern asan and compiler to spot more issues, but not a blocker